tgui/packages/tgui/styles/interfaces/DecalPainter.scss

```diff
@@ -0,0 +1,9 @@
+.DecalPainter__CellButton {
+  margin: 2px !important;
+  padding: 0.5em !important;
+}
+
+.DecalPainter__CellIcon {
+  display: block !important;
+  filter: drop-shadow(0px 0px 0px 2px black);
```

Comment:
Why this and not box-shadow?

Reply:
Ya know what? That dates back to me attempting a different style and I can just remove that as I found it doesn't even work in IE.

Reply:
Removed.